test/test_alloc.cpp

```diff
@@ -242,7 +242,7 @@ TEST(Alloc_BadBuffer)
     GROUP_TEST_PATH(path);
 
     // Produce an invalid buffer
-    char buffer[32];
+    alignas(8) char buffer[32];
```

Comment:
UBSAN was complaining about this while testing. The buffer gets reinterpret_cast'd into a pointer to a struct while trying to validate it as a realm buffer, so that storage needs to be aligned.